scripts/docker/install_graph_ds_lib.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+CONF="$NEO4J_HOME"/conf/neo4j.conf
+
+curl -k -LO https://s3-eu-west-1.amazonaws.com/com.neo4j.graphalgorithms.dist/graph-data-science/neo4j-graph-data-science-1.1.0-standalone.zip
```

Comment:
Yeah it's not necessary right now but I figured it may be in the future so I just left there as a reference (figuring how to install it from the docs wasn't easy so I'm afraid I may forget)